editor/app/src/main/java/ca/ubc/eml/soiltopargraphy/editor/ui/main/MainFragment.kt

```diff
@@ -16,15 +19,27 @@ class MainFragment : Fragment() {
 
     private lateinit var viewModel: MainViewModel
 
+    fun onCreateInfoButtonClick(view: View){
```

Comment:
```suggestion
    private fun onCreateInfoButtonClick(view: View){
```
nitpick: use a private function